openshift/deploy/webserver.yml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: spm-engine-webserver
+
+parameters:
+  - name: IMAGE_NAMESPACE
+    displayName: Image namespace
+  - name: IMAGE_TAG
+    displayName: Image tag
+    value: latest
+objects:
+  - apiVersion: v1
+    kind: DeploymentConfig
+    metadata:
+      labels: { app: spm-engine }
+      name: spm-engine-webserver
+    spec:
+      replicas: 1
+      selector:
+        app: spm-engine
+        deploymentconfig: spm-engine-webserver
+      template:
+        metadata:
+          labels:
+            app: spm-engine
+            deploymentconfig: spm-engine-webserver
+          name: spm-engine-webserver
+        spec:
+          containers:
+            - image: ${IMAGE_NAMESPACE}/spm-engine-webserver:${IMAGE_TAG}
+              imagePullPolicy: Always
+              name: spm-engine-webserver
+              command: [ ./webserver/entrypoint.sh ]
+              resources:
+                limits: { cpu: 500m,  memory: 512Mi }
+                requests: { cpu: 200m, memory: 512Mi }
+              env:
+                - { name: POSTGRESQL_HOST, value: spm-engine-database }
+                - { name: POSTGRESQL_USER, value: admin }
+                - { name: POSTGRESQL_PASSWORD, value: passwd }
+                - { name: POSTGRESQL_DATABASE, value: spm }
+          dnsPolicy: ClusterFirst
```

Comment:
readinessProbe and livenessProbe are missing

Reply:
Fixed